lib/helpers.js

```diff
@@ -69,6 +69,10 @@ export const queriedForFieldsOtherThanBlacklisted = (fieldASTs, blacklistedField
   const queriedFields = map(flatMap(fieldASTs, "selectionSet.selections"), "name.value")
   return difference(queriedFields, blacklistedFields).length > 0
 }
+export const queryContainsField = (fieldASTs, soughtField) => {
+  const queriedFields = map(flatMap(fieldASTs, "selectionSet.selections"), "name.value")
```

Comment:
Extract this into a function and share it between `queryContainsField` and `queriedForFieldsOtherThanBlacklisted`.